src/jaguar.toit

```diff
@@ -127,14 +132,19 @@ serve arguments:
         backoff := Duration --s=5
         logger.info "backing off for $backoff"
         sleep backoff
-  Task.group lambdas
+  if lambdas.size == 1:
+    lambdas[0].call
+  else:
+    Task.group lambdas
```

Comment:
I wonder if `Task.group` should just handle the special case of 0 and 1 lambdas passed to it? Not for now.